docs/banzai_nres/index.rst

```diff
@@ -8,3 +8,135 @@ Reference/API
 =============
 
 .. automodapi:: banzai_nres
+
+
+
+Running Code Tests Locally
+==========================
+
+To test code style: "tox -e codestyle"
+To test documentation: "tox -e build_docs"
+To test the functionality of the code itself (on python 3.8): "tox -e py38-test-alldeps"
+
+Output Data Products
+====================
+
+Here we cover the various quantities contained in the banzai-nres output data products.
+
+
+Master wavelength calibration files
+-----------------------------------
+
+Master wavelength calibrations are stacked arc (double) exposures, wavelength calibrated primarily
+with xwavecal routines. These file names follow the convention `site-camera-date-double-binmode-litfibers.fits.fz`
```

Comment:
This feels kind of like non sequitur. xwavecal doesn't really use this file naming convention. This is an LCO thing.